arangod/RocksDBEngine/RocksDBMetaCollection.cpp

```diff
@@ -1715,6 +1716,31 @@ ErrorCode RocksDBMetaCollection::doLock(double timeout, AccessMode::Type mode) {
       std::chrono::duration<double>(timeout));
 
   bool gotLock = false;
+  if (timeout_us > std::chrono::microseconds(1000000)) {
```

Comment:
nit: the value `std::chrono::microseconds(1000000)` is used 4 times in this code. IMHO it would be better to put the threshold value into a constexpr once, and then refer to it only by its symbol name.